src/index.js

```diff
@@ -54,7 +56,7 @@ const getUnitValue = (theme, cssProp, n) => {
 
   const val = get(theme, [area, n], n);
 
-  return px(val);
+  return numValProps.includes(cssProp) ? val : px(val);
```

Comment:
[includes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes) is not supported in IE 

we don't want to add polyfills to keep this file small